fedimint-api/src/db/mod.rs

```diff
@@ -56,16 +59,58 @@ pub trait DatabaseValue: Sized + SerializableDatabaseValue {
 }
 
 pub type PrefixIter<'a> = Box<dyn Iterator<Item = Result<(Vec<u8>, Vec<u8>)>> + Send + 'a>;
+type Notifications = DashMap<Box<[u8]>, Arc<Notify>>;
```

Comment:
Comments, comments. Please comments. :) . At least one line of docstring on every struct and major type and most important methods.